main.nf

```diff
@@ -222,40 +222,44 @@ process alevin {
         set val(runId), file("${runId}_ALEVIN_fry_quant") into ALEVIN_RESULTS
         set val(runId), file("${runId}_ALEVIN_fry_map/aux_info/meta_info.json") into ALEVIN_STATS
 
-    
-    """
-    
-    salmon alevin ${barcodeConfig} --sketch -1 \$(ls barcodes*.fastq.gz | tr '\\n' ' ') -2 \$(ls cdna*.fastq.gz | tr '\\n' ' ') \
-        -i ${transcriptomeIndex} -p ${task.cpus} -o ${runId}_ALEVIN_fry_map 
-
-    if [ "${params.protocol}" = "10xv2" ]
-    then
-        alevin-fry generate-permit-list --input ${runId}_ALEVIN_fry_map -d fw --unfiltered-pl ${baseDir}/whitelist/737K-august-2016.txt --output-dir ${runId}_ALEVIN_fry_quant_tmp --min-reads 10
-    elif [ "${params.protocol}" = "10xv3" ]
-    then
-        alevin-fry generate-permit-list --input ${runId}_ALEVIN_fry_map -d fw --unfiltered-pl ${params.10xv3.whitelist} --output-dir ${runId}_ALEVIN_fry_quant_tmp --min-reads 10
-    elif [ "${params.protocol}" = "10x5prime" ]
-    then
-        alevin-fry generate-permit-list --input ${runId}_ALEVIN_fry_map -d rc  --output-dir ${runId}_ALEVIN_fry_quant_tmp --force-cells 100000 --min-reads 10
-    else
-        alevin-fry generate-permit-list --input ${runId}_ALEVIN_fry_map -d fw --output-dir ${runId}_ALEVIN_fry_quant_tmp  --force-cells 100000 --min-reads 10
-    fi
+    script:
```

Comment:
This is new.  Probably without the 2 parameters you've defined it wasn't necessary